app/views/devise/registrations/edit.html.erb

```diff
@@ -1,43 +1,61 @@
-<h2>Edit <%= resource_name.to_s.humanize %></h2>
+<div class="flex items-center justify-center" >
+  <h2>Edit <%= resource_name.to_s.humanize %></h2>
+</div>
 
-<%= form_for(resource, as: resource_name, url: registration_path(resource_name), html: { method: :put }) do |f| %>
-  <%= render "devise/shared/error_messages", resource: resource %>
-
-  <div class="field">
-    <%= f.label :email %><br />
-    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
-  </div>
-
-  <% if devise_mapping.confirmable? && resource.pending_reconfirmation? %>
-    <div>Currently waiting confirmation for: <%= resource.unconfirmed_email %></div>
-  <% end %>
-
-  <div class="field">
-    <%= f.label :password %> <i>(leave blank if you don't want to change it)</i><br />
-    <%= f.password_field :password, autocomplete: "new-password" %>
-    <% if @minimum_password_length %>
-      <br />
-      <em><%= @minimum_password_length %> characters minimum</em>
-    <% end %>
-  </div>
-
-  <div class="field">
-    <%= f.label :password_confirmation %><br />
-    <%= f.password_field :password_confirmation, autocomplete: "new-password" %>
-  </div>
 
-  <div class="field">
-    <%= f.label :current_password %> <i>(we need your current password to confirm your changes)</i><br />
-    <%= f.password_field :current_password, autocomplete: "current-password" %>
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name), html: { method: :put }) do |f| %>
+  
+  <div class="flex items-center justify-center">  
+    <div class="flex flex-col items-center">
+      <%= render "devise/shared/error_messages", resource: resource %>
+      
+      <div>
+        <%= f.label :username, class: 'font-bold text-x'%><br />
+        <%= f.text_field :username, autofocus: true, class: 'rounded mb-2' %>
+      </div>
+
+      <div>
+        <%= f.label :email, class: 'font-bold text-x'  %><br />
+        <%= f.email_field :email, autofocus: true, autocomplete: "email", class: 'rounded mb-2'  %>
+      </div>
+
+      <% if devise_mapping.confirmable? && resource.pending_reconfirmation? %>
+        <div>Currently waiting confirmation for: <%= resource.unconfirmed_email %></div>
+      <% end %>
+
+      <div>
+        <%= f.label :password, class: 'font-bold text-x' %> <i>(leave blank if you don't want to change it)</i><br />
+      </div>
+      <div>
+        <%= f.password_field :password, autocomplete: "new-password", class: 'rounded mb-2'  %>
+        <% if @minimum_password_length %>
+        <% end %>
```

Comment:
I think that this condition doesn't have any purpose, or what is the reason for it?

Reply:
fixed!!